middlewares/client/setting.middlewares.js

```diff
@@ -0,0 +1,6 @@
+const SettingGeneral = require("../../models/setting-general.model")
+module.exports.settingGeneral = async (req,res,next) =>{
+    const settingGeneral = await SettingGeneral.findOne({});
```

Comment:
This is 1 extra database query for each request. One query might sound small but they add up quickly. Can you please add a simply in-memory cache here so that the result of `SettingGeneral.findOne({})` is cached for 1 minute? It means 1st request will still query database, but then subsequent requests in the next minute will just read the data from memory instead of querying database.